MarsAdvancedTaskPart1NUnitAutomation/Test/LanguageTest.cs

```diff
@@ -0,0 +1,241 @@
+﻿using MarsAdvancedTaskPart1NUnitAutomation.DataModel;
+using MarsAdvancedTaskPart1NUnitAutomation.Pages.ProfileOverview.ProfileNavigationMenuComponents;
+using MarsAdvancedTaskPart1NUnitAutomation.ReportClass;
+using MarsAdvancedTaskPart1NUnitAutomation.Utilities;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsAdvancedTaskPart1NUnitAutomation.Test
+{
+    
+    [TestFixture]
+    public class LanguageTest : CommonDriver
+    {
+
+        List<LanguageDM> languageList;
+        LanguageFeatures languageFeaturesObj;
+        ProfileNavigationTabs profileNavigationTabsObj;
+        GenerateReport? generateReport;
+        public LanguageTest()
+        {
+            languageList = new List<LanguageDM>();
+            languageFeaturesObj = new LanguageFeatures();
+            profileNavigationTabsObj = new ProfileNavigationTabs();
+        }
+
+
+        [OneTimeSetUp]
+        public void ReportAndJSONDataMethod()
+        {
+            generateReport = new GenerateReport();
+            generateReport?.GenerateExtentReport(@"Reports\LanguageExtentReport.html");
```

Comment:
You should not need to have the onetime setup in each test as it's only needed to be added once please change this